test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -3,19 +3,31 @@
  */
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
+test.use( {
+	navBlockUtils: async ( { page, requestUtils }, use ) => {
+		await use( new NavigationBlockUtils( { page, requestUtils } ) );
+	},
+} );
+
 test.describe(
 	'As a user I want the navigation block to fallback to the best possible default',
 	() => {
 		test.beforeAll( async ( { requestUtils } ) => {
-			await requestUtils.activateTheme( 'emptytheme' );
+			await requestUtils.activateTheme( 'twentytwentythree' );
```

Comment:
Any reason why we want to use TT3 here?

Reply:
because `emptytheme` won't show the navigation block unless we navigate to the specific post. It seemed like the simpler solution.

Reply:
Maybe a comment above will make it easier to read? 🙂